test/e2e/lib/components/notices-component.js

```diff
@@ -15,18 +15,52 @@ import AsyncBaseContainer from '../async-base-container';
 
 export default class NoticesComponent extends AsyncBaseContainer {
 	constructor( driver ) {
-		super( driver, By.css( '#notices' ), null, config.get( 'explicitWaitMS' ) * 3 );
+		super( driver, By.css( '.wpcom-site' ), null, config.get( 'explicitWaitMS' ) * 3 );
 	}
 
-	async inviteMessageTitle() {
-		const selector = By.css( '.invites__title' );
+	async isSuccessNoticeDisplayed( click = false ) {
+		const selector = By.css( '.notice.is-success' );
+		const actionSelector = By.css( '.notice.is-success a' );
+		const isDisplayed = await driverHelper.waitTillPresentAndDisplayed( this.driver, selector );
+		if ( click === true ) {
+			await driverHelper.clickWhenClickable( this.driver, actionSelector );
+		}
+		return isDisplayed;
+	}
+
+	async isErrorNoticeDisplayed() {
+		const selector = By.css( '.notice.is-error' );
+		return await driverHelper.isEventuallyPresentAndDisplayed( this.driver, selector );
+	}
+
+	async getNoticeContent() {
+		const selector = By.css( '.notice .notice__text' );
 		await driverHelper.waitTillPresentAndDisplayed( this.driver, selector );
 		return await this.driver.findElement( selector ).getText();
 	}
 
-	async followMessageTitle() {
-		const selector = By.css( '#notices .notice__text' );
+	async dismissNotice() {
+		const selector = By.css( '.notice.is-dismissable .notice__dismiss' );
+		await driverHelper.waitTillPresentAndDisplayed( this.driver, selector );
+		return await driverHelper.clickWhenClickable( this.driver, selector );
+	}
+
+	async getInviteNoticeContent() {
+		const selector = By.css( '.notice .invites__title' );
 		await driverHelper.waitTillPresentAndDisplayed( this.driver, selector );
 		return await this.driver.findElement( selector ).getText();
 	}
+
+	async isNoticeDisplayed( click = false ) {
```

Comment:
Same as for `isSuccessNoticeDisplayed`. Also, this method seems quite similar to `isSuccessNoticeDisplayed`, do you think we can extract common pieces into a single method?

Reply:
`isNoticeDisplayed` is for info notice, and `isSuccessNoticeDisplayed` is for success notice. It looked too complicated to put it all in one method. Do you have a proposal on how to do it? I agree, it would be better to put it all in one method.

Reply:
> Same as for isSuccessNoticeDisplayed

I mean the part about `isEventuallyPresentAndDisplayed` vs `waitTillPresentAndDisplayed`. If we switch to use the same function we can do something like this: 


```
async _isNoticeDisplayed( selector, actionSelector, click = false ) {
		const isDisplayed = await driverHelper.isEventuallyPresentAndDisplayed( this.driver, selector );
		if ( click === true ) {
			await driverHelper.clickWhenClickable( this.driver, actionSelector );
		}
		return isDisplayed;
	}

async isSuccessNoticeDisplayed( click = false ) {
		const selector = By.css( '.notice.is-success' );
		const actionSelector = By.css( '.notice.is-success a' );
		return await this._isNoticeDisplayed( selector, actionSelector, click);
	}

async isNoticeDisplayed( click = false ) {
		const selector = By.css( '.notice' );
		const actionSelector = By.css( '.notice a' );
		return await this._isNoticeDisplayed( selector, actionSelector, click);
	}
```

Reply:
Nice! Done.